src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Again, the file name should be `star_actived.png` instead of `star-active.svg`. Ensure that the file names are consistent with the provided file structure.